octo_client/client.py

```diff
@@ -147,18 +148,20 @@ def _make_request(
         return response_json
 
     def _filter_request_log_data(
-            self, data_length: int, content: Union[str, dict]
+        self, data_length: int, content: Union[str, dict]
     ) -> Optional[Union[str, dict]]:
         if self.log_requests:
+            content = hide_sensitive_data(content)
             if self.log_size_limit and data_length > self.log_size_limit:
                 return "TRUNCATED"
             return content
         return None
 
     def _filter_log_data(
-            self, response_length: int, response_content: Union[str, dict]
+        self, response_length: int, response_content: Union[str, dict]
     ) -> Optional[Union[str, dict]]:
         if self.log_responses:
+            response_content = hide_sensitive_data(response_content)
```

Comment:
**issue:** we need to create a copy of the response otherwise the `hide_sensitive_data` method will modify the real response instead of only the log entry.